.github/workflows/python-CI.yml

```diff
@@ -109,6 +109,7 @@ jobs:
             matrix:
                 os: [macos-12, windows-latest, ubuntu-latest]
                 python-version: ["3.8"]
+            fail-fast: false
```

Comment:
we sure we want to keep this?

Reply:
I personally like this, but will ask the group.